packages/components/pagination/src/pagination.ts

```diff
@@ -287,6 +289,14 @@ export default defineComponent({
       if (currentPageBridge.value > val) currentPageBridge.value = val
     })
 
+    watch(
+      [pageSizeBridge, currentPageBridge],
+      (value) => {
+        emit('change', ...value)
+      },
+      { flush: 'post' }
+    )
```

Comment:
I think currentPage should be before pageSize, it uses a higher frequency than pageSize.
What do you think

Reply:
You are right.